api/src/main/resources/workbench.yaml

```diff
@@ -935,6 +935,23 @@ paths:
             $ref: "#/definitions/CriteriaListResponse"
 
   # Cohort Review  #######################################################################
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/participants/filters:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+      - $ref: '#/parameters/cohortId'
+      - $ref: '#/parameters/cdrVersionId'
+    get:
+      tags:
+        - cohortReview
+      description: Will return a list all values for gender, race and ethnicity.
+      operationId: "getParticipantFilters"
+      responses:
+        200:
+          description: A cohortReviewId and cohort count
```

Comment:
This description is outdated (probably copy/pasted?)

Reply:
yep.. done.